src/App.vue

```diff
@@ -12,6 +12,17 @@
   </v-app>
 </template>
 
+<script setup>
+import { useTheme } from 'vuetify'
+
+const theme = useTheme()
+
+const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)')
```

Comment:
easy to add a settings toggle in a following PR ?
I know some users will be asking for it to revert back to the light mode ^^

Reply:
in https://hunger.openfoodfacts.org its interesting because the settings are not "hidden" behind auth

Reply:
it could be a header (or footer) toggle. vs a new settings. or move the settings outside of the auth wall ? :thinking:

Reply:
I'll do it in this PR, just wanted an insight on where to put the toggle :)

I do not think it should require login, and I don't know if I want to tackle moving the setting page :smile: 

The Footer has a lot of available real estate, so I guess I'll put something there for now ?

Reply:
Footer sounds great !